### PR TITLE
Internationalization

### PR DESCRIPTION
**Pull request by arctouch-vivyanrodrigues**
**Description**
- [x] Add i18n for English language

**Ticket**
AWE-26 - Internationalization

### CODE REVIEW

File: lib/l10n/intl_en.arb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "sliderTitle1": "Welcome to ArcWeather",
+  "sliderTitle2": "Follow different places",
+  "sliderTitle3": "Never forget your umbrella",
+  "sliderText1": "Your new favorite weather app, with all the features you need to have the best information available.",
+  "sliderText2": "You can easily add as many different locations as you wish.",
+  "sliderText3": "Accurate weather forecast that you can trust to never get caught in the rain again.",
+  "choseTemperature": "Choose your preferred temperature scale:",
+  "celcius": "°C",
+  "fahrenheit": "°F",
+  "empty": ""
```
Comment (arctouch-victorbotamedi) [inline lib/l10n/intl_en.arb:20]: i don't get this one. Empty is empty 🤔

File: lib/screens/onboarding_screen.dart [hunk 2]
```diff
@@ -20,24 +21,9 @@ class Onboarding extends StatefulWidget {
 class _OnboardingState extends State<Onboarding> with TickerProviderStateMixin {
   static const onboardingCurve = Cubic(0.175, 0.885, 1.0, 1.1);
   late OnboardingStore _onboardingStore;
+  late AppLocalizations s;
```
Comment (arctouch-victorbotamedi) [inline lib/screens/onboarding_screen.dart:24]: You shouldn't have this on your widget.

File: lib/screens/onboarding_screen.dart [hunk 3]
```diff
@@ -66,7 +52,22 @@ class _OnboardingState extends State<Onboarding> with TickerProviderStateMixin {
 
   @override
   Widget build(BuildContext context) {
+    s = AppLocalizations.of(context)!;
```
Comment (arctouch-victorbotamedi) [inline lib/screens/onboarding_screen.dart:55]: this should be a local variable

File: pubspec.yaml [hunk 4]
```diff
@@ -41,4 +46,24 @@ flutter:
     - family: Avenir Light
       fonts:
         - asset: assets/fonts/AvenirLight.otf
-          weight: 900
\ No newline at end of file
+          weight: 900
+
+flutter_intl:
```
Comment (arctouch-victorbotamedi) [inline pubspec.yaml:51]: i think you have conflicting configurations, you set a l10n.yml but also have configurations here. Make sure to configure all in the same place

File: lib/l10n/intl_en.arb (hunk 1 above)
Comment (arctouch-vivyanrodrigues) [reply lib/l10n/intl_en.arb:20]: Done!

File: lib/screens/onboarding_screen.dart (hunk 2 above)
Comment (arctouch-vivyanrodrigues) [reply lib/screens/onboarding_screen.dart:24]: Done!

File: lib/screens/onboarding_screen.dart (hunk 3 above)
Comment (arctouch-vivyanrodrigues) [reply lib/screens/onboarding_screen.dart:55]: Done!

File: pubspec.yaml (hunk 4 above)
Comment (arctouch-vivyanrodrigues) [reply pubspec.yaml:51]: Done!

**Review by arctouch-victorbotamedi — APPROVED** [review]

**Pull request merged**